tasks/main.yml

```diff
@@ -86,7 +86,7 @@
           hosted-engine --set-maintenance --mode=none
 
     - name: Get VM list
-      ovirt_vm_facts:
+      ovirt_vm_info:
```

Comment:
I'm probably missing something here, because you are VMs information, but you are not using anywhere. Just be aware that calling ovirt_vm_info doesn't store information about VMs into ansible_facts, so if you really need those information, then you to register a variable to store results of ovirt_vm_info:

https://docs.ansible.com/ansible/latest/modules/ovirt_vm_info_module.html#examples

Reply:
Yes exactly, here is another example https://github.com/oVirt/ovirt-ansible-cluster-upgrade/pull/59/files.

Reply:
We are fetching the list of VMs to shutdown them, 
see: https://github.com/oVirt/ovirt-ansible-shutdown-env/blob/master/tasks/main.yml#L102

On ansible 2.7/2.8 we were using ovirt_vm_facts that was implicitly registering ovirt_vms fact ( https://docs.ansible.com/ansible/2.7/modules/ovirt_vm_facts_module.html#notes ). ovirt_vm_info doesn't